src/components/DataSession/DataSession.vue

```diff
@@ -27,54 +28,23 @@ async function addOperation(operationDefinition) {
 	await fetchApiCall({url: url, method: 'POST', body: operationDefinition, successCallback: emit('reloadSession'), failCallback: handleError})
 }
 
-const saveImages = (data) => {
-	const results = data.results
-	if (results.length) {
-		images.value.push(data.results[0])
-	}
-}
-
-const getImages = async () => {
-	const responseData = props.data
-	const inputData = responseData.input_data
-	for (const data of inputData) {
-		const basename = data.basename
-		const url =  `https://datalab-archive.photonranch.org/frames/?basename_exact=${basename}-small`
-		await fetchApiCall({url: url, method: 'GET', successCallback: saveImages, failCallback: handleError})
-	}
-}
-
-onMounted(() => {
-	getImages()
-})
-
 </script>
 
 <template>
   <v-container class="d-lg-flex ds-container">
-    <v-row v-if="images.length">
-      <v-col
-        v-for="image of images"
-        :key="image.basename"
-        :cols="calculateColumnSpan(images.length, 4)"
-      >
-        <v-img
-          :src="image.url"
-          :alt="image.basename"
-          cover
-          aspect-ratio="1"
-        />
-      </v-col>
-    </v-row>
+    <image-grid
+      :data="data"
+      :column-span="calculateColumnSpan(data.input_data.length, 4)"
```

Comment:
magic number?

Reply:
The magic number in this case is 4, its magic because its not easily apparent to a reader of the code what 4 is representing. This SO explains them in more detail. 
https://stackoverflow.com/questions/47882/what-are-magic-numbers-and-why-do-some-consider-them-bad

Reply:
Done! Thank you : ) I added both magic numbers in both cases